collect_app/src/main/java/org/odk/collect/android/configure/qr/QRCodeScannerFragment.kt

```diff
@@ -87,4 +87,8 @@ class QRCodeScannerFragment : BarCodeScannerFragment() {
     override fun isQrOnly(): Boolean {
         return true
     }
+
+    override fun shouldConfirm(): Boolean {
```

Comment:
I'd far prefer to not be doing configuration via inheritance here, but my guess is that we'll want to break this hierarchy apart soon so that the design of the "reconfigure" QR code scanner is more similar to the one that adds projects. It'll be better to do that work when making actual changes.